OADP-5784: Release Notes for OADP 1.4.5 (#92744)
openshift/openshift-docs: modules/oadp-1-4-5-release-notes.adoc

// Module included in the following assemblies:
//
// * backup_and_restore/oadp-1-4-release-notes.adoc

:_mod-docs-content-type: REFERENCE

[id="oadp-1-4-5-release-notes_{context}"]
= {oadp-short} 1.4.5 release notes

{oadp-first} 1.4.5 is a Container Grade Only (CGO) release, which is released to refresh the health grades of the containers. No code was changed in the product itself compared to that of {oadp-short} 1.4.4.

The following CVEs have been fixed in {oadp-short} 1.4.5: 

* link:https://access.redhat.com/security/cve/CVE-2024-45337[CVE-2024-45337] 

* link:https://access.redhat.com/security/cve/CVE-2024-45338[CVE-2024-45338] 

* link:https://access.redhat.com/security/cve/CVE-2025-21613[CVE-2025-21613]

* link:https://access.redhat.com/security/cve/CVE-2025-27144[CVE-2025-27144]

* link:https://access.redhat.com/security/cve/cve-2025-22868[CVE-2025-22868] 

* link:https://access.redhat.com/security/cve/CVE-2025-22869[CVE-2025-22869] 

> * link:https://access.redhat.com/security/cve/CVE-2025-30204[CVE-2025-30204] 

Reviewer: @anarnold97 , two things that come to mind

1. Didn't something change that no longer require providing CVE fixes in release notes?

2. The mustgather image has been refactored ([OADP-5904](https://issues.redhat.com/browse/OADP-5904)).  Similar to what was done for 1.5 in https://github.com/openshift/openshift-docs/pull/93476.  This closed PR https://github.com/openshift/openshift-docs/pull/93488 might already have everything needed to cover it.  Worth mentioning the change and updating the docs like was done for 1.5?

Author: @rayfordj - yeah but Wes asked about including CVEs. And some teams include the CVEs:

<img width="1563" height="206" alt="image" src="https://github.com/user-attachments/assets/8a39baa7-aa41-41bf-95c5-56f0eb9b6fdb" />

But they do not include a link to the JIRA ticket describing the CVE, which is usually set to `Red Hat Employee Only`, so I thought this might work

As for `must-gather`, not a problem

Reviewer: ah, cool, no objection from me _including_ or _not including_ CVEs. :sunglasses:

// Module included in the following assemblies:
//
// * backup_and_restore/oadp-1-4-release-notes.adoc

:_mod-docs-content-type: REFERENCE

[id="oadp-1-4-5-release-notes_{context}"]
= {oadp-short} 1.4.5 release notes

> 

Author: ```suggestion

The {oadp-first} 1.4.5 release notes lists new features and resolved issues.

```

[id="new-features-1-4-5_{context}"]
== New features

.Collecting logs with the `must-gather` tool has been improved with a Markdown summary

> You can collect logs, and information about {oadp-first} custom resources by using the `must-gather` tool. The `must-gather` data must be attached to all customer cases.

Reviewer 2: Remove the comma:
```suggestion
You can collect logs and information about {oadp-first} custom resources by using the `must-gather` tool. The `must-gather` data must be attached to all customer cases.
```

> You can collect logs, and information about {oadp-first} custom resources by using the `must-gather` tool. The `must-gather` data must be attached to all customer cases.

Reviewer 2: [Change to active voice](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice):
```suggestion
You can collect logs, and information about {oadp-first} custom resources by using the `must-gather` tool. You must attach the `must-gather` data to all customer cases.
```

> This tool generates a Markdown output file with the collected information, which is located in the `must-gather` logs clusters directory.

Author: ```suggestion
This tool generates a Markdown output file with the collected information, which is located in the `must-gather` logs clusters directory. link:https://issues.redhat.com/browse/OADP-5904[(OADP-5904)]
```

> This tool generates a Markdown output file with the collected information, which is located in the `must-gather` logs clusters directory. link:https://issues.redhat.com/browse/OADP-5904[(OADP-5904)]

Reviewer 2: Suggestion for clarity: 
```suggestion
This tool generates a Markdown output file with the collected information, which is located in the clusters directory of the `must-gather` logs  link:https://issues.redhat.com/browse/OADP-5904[(OADP-5904)]
```

// Module included in the following assemblies:
//
// * backup_and_restore/oadp-1-4-release-notes.adoc

:_mod-docs-content-type: REFERENCE

[id="oadp-1-4-5-release-notes_{context}"]
= {oadp-short} 1.4.5 release notes

The {oadp-first} 1.4.5 release notes lists new features and resolved issues.

[id="new-features-1-4-5_{context}"]
== New features

.Collecting logs with the `must-gather` tool has been improved with a Markdown summary

You can collect logs and information about {oadp-first} custom resources by using the `must-gather` tool. The `must-gather` data must be attached to all customer cases.

> This tool generates a Markdown output file with the collected information, which is located in the clusters directory of the `must-gather` logs  link:https://issues.redhat.com/browse/OADP-5904[(OADP-5904)]

Author: ```suggestion
This tool generates a Markdown output file with the collected information, which is located in the clusters directory of the `must-gather` logs.  link:https://issues.redhat.com/browse/OADP-5904[(OADP-5904)]
```
